package.json

```
@@ -3,10 +3,16 @@
   "version": "0.2.0",
   "description": "Embed BPMN 2.0 diagrams in your React app",
   "main": "dist/index.js",
+  "files": [
+    "dist",
+    "src"
+  ],
   "scripts": {
     "all": "npm run bundle",
     "bundle": "NODE_ENV=production rollup -c",
-    "prepublishOnly": "npm run bundle"
+    "prepublishOnly": "npm run bundle",
+    "postinstall": "cd example && npm install",
+    "start:example": "cd example && npm start"
```

As we don't have an example, we could use the `start` script directly
```suggestion
    "start": "cd example && npm start"
```